.bazelrc

```diff
@@ -15,3 +15,10 @@ test:lint --test_tag_filters=lint
 # you can run non-lint tests with:
 # bazel test //... --config=unit
 test:unit --test_tag_filters=-lint
+
+build:remote --remote_cache=remotebuildexecution.googleapis.com
+build:remote --tls_enabled=true
+build:remote --remote_timeout=3600
+build:remote --auth_enabled=true
+
+build:remote-fejta --remote_instance_name=projects/fejta-prod/instances/default_instance
```

Comment:
Use `bazel foo --config=remote-fejta` to include these values (only useful when used in addition to `--config=remote`)

Reply:
Can this rule include `--config=remote` to automatically include the args above when just `--config=remote-fejta` is specified?

Reply:
this looks like your personal settings are living in a file shared by all other developers, which seems like an open source no-no... am I misunderstanding? is there a way to do this without committing your changes to everyone else's copy of this repo?

Reply:
>  automatically include the args

One config does not extend another. I prefer this composition over copying all the rules in two locations.

> am I misunderstanding?

We have all sorts of concrete config in this repo, see https://github.com/kubernetes/test-infra/blob/master/hack/print-workspace-status.sh#L25 https://github.com/kubernetes/test-infra/blob/master/prow/cmd/build/BUILD.bazel#L7 etc

What are you concerned about exactly? This is useful for people to help test this optional feature, rather than asking them to type out the super long name each time. I would expect that once it proves itself more useful we'd replace it with something more general, ideally something open to all community members.

Reply:
This particular line makes it look like this is your personal cluster, rather than a shared resource that anyone can use.  I feel the same way about the `BUILD.bazel` line you link.  It reminds of an anti-pattern where someone treats a shared codebase as if it was their personal pet project, and can look bad if that individual is a lead on the project.

I feel like this probably isn't the case, given your "we'd replace it with something more general" line.  None of this intent is communicated in the name though, nor a comment nearby.

If the name made it clearer this was just a demo, trial, or an experiment, I probably would have viewed this differently.  The `print-workspace-status.sh` example you link fits this pattern.

Having said my piece I'll get off of my soap box and put the lgtm back.

Reply:
Understood, this is not intended to be permanent, just while we're exploring the idea.